src/App.jsx

```diff
@@ -62,6 +92,7 @@ function App() {
           fields: {
             title: newTodo.title,
             isCompleted: false,
+            createdTime: new Date().toISOString().split('T')[0]
```

Comment:
`createdTime` should be facilitated by the database. Client should not handle this. This could also lead to some error in data synchronization .